aait/backend/group-2A/Application/Features/Comment/Handlers/Queries/GetCommentsByPostIdRequestHandler.cs

```diff
@@ -23,7 +24,7 @@ public async Task<List<CommentDto>> Handle(GetCommentsByPostIdRequest request, C
 
         if (comments == null)
         {
-            throw new Exception("No comments");
+            throw new NotFoundException(nameof(Domain.Entities.Comment), request.PostId);
         }
         
         return _mapper.Map<List<CommentDto>>(comments);
```

Comment:
instead of returning the comment object . you can return the response object you have defined